mlrun/api/api/endpoints/frontend_spec.py

```diff
@@ -44,7 +45,7 @@ def get_frontend_spec(
 ):
     jobs_dashboard_url = None
     session = auth_info.session or session
-    if session:
+    if session and (not auth_info.session and is_iguazio_session_cookie(session)):
```

Comment:
why `not auth_info.session`? we might get that session while in iguazio

Reply:
correct, updated the flow to assume iguazio session/cookie session